ma_gym/envs/traffic_junction/traffic_junction.py

```diff
@@ -88,9 +213,29 @@ def get_agent_obs(self):
             _agent_i_obs = [0 for _ in range(self.n_agents)]
             _agent_i_obs[agent_i] = 1
 
+            # gets other agents position relative to agent_i position, 3x3 view mask
+            _other_agents_pos = np.zeros(self._agent_view_mask)  # other agents location in neighbour
+
+            for row in range(max(0, pos[0] - 1), min(pos[0] + 1 + 1, self._grid_shape[0])):
+                for col in range(max(0, pos[1] - 1), min(pos[1] + 1 + 1, self._grid_shape[1])):
+                    if PRE_IDS['agent'] in self._full_obs[row][col]:
+                        # get relative position for other agents location
+                        _other_agents_pos[row - (pos[0] - 1), col - (pos[1] - 1)] = 1
+
+            # set center value to 0 that belongs to agent_i TODO: see if this stays or not
```

Comment:
Is this `Todo` done?

Reply:
While you are at it, can you also clean up other `Todo` which are done?

Reply:
Sure will clean that